tools/copy_production_db_to_staging

```diff
@@ -0,0 +1,158 @@
+            --query "DBInstances[0].VpcSecurityGroups[0].VpcSecurityGroupId" \
+            --output text)"
+
+        # Remove rds: from snapshot ID & use as database ID
+        NEW_DATABASE_ID="${SNAPSHOT_ID:4}"
+
+        RESTORE="aws rds restore-db-instance-from-db-snapshot \
+            --no-paginate \
+            --db-instance-identifier $NEW_DATABASE_ID \
+            --db-snapshot-identifier $SNAPSHOT_ID \
+            --db-instance-class $STAGING_RDS_INSTANCE_TYPE \
+            --db-subnet-group-name $STAGING_RDS_SUBNET_GROUP\
+            --vpc-security-group-ids $STAGING_RDS_VPC_SECURITY_GROUP"
+
+        echo "Will run '$RESTORE'"
+        read -n 1 -s -r -p "Press any key to begin"
+        echo
+
+        $RESTORE
+
+        # Restore will complete right away, but we can't begin to query
+        # the database until the status is "available"
+        CHECK_STATUS="aws rds describe-db-instances \
+            --db-instance-identifier $NEW_DATABASE_ID \
+            --query DBInstances[0].DBInstanceStatus --output text"
+
+        echo -n "Polling until new instance is ready"
+        while [ "$($CHECK_STATUS)" != 'available' ]; do
+            echo -n "."
+            sleep 10
+        done
+        echo
+
+        NEW_DATABASE_ENDPOINT="$(aws rds describe-db-instances \
+            --db-instance-identifier $NEW_DATABASE_ID \
+            --query DBInstances[0].Endpoint.Address --output text)"
+
+        pushd "$PROJECT_ROOT/load-tests/anonymize-the-database"
+
+        echo "Installing requirements for anonymization script"
+        python3 -m venv ./venv
+        source ./venv/bin/activate
+        pip install -r requirements.txt
+
+        ANONYMIZATION_QUERIES="$(PGPASSWORD=$PRODUCTION_PGPASSWORD PGHOST=$NEW_DATABASE_ENDPOINT \
+                                 PGPORT=5432 ALLOWED_EMAILS=$ALLOWED_EMAILS \
+                                 ./generate_anon_queries)"
+
+        popd
+
+        read -n 1 -s -r -p "Press any key to start anonymizing the database"
+        echo
+        export PGPASSWORD="$PRODUCTION_PGPASSWORD" 
+        echo "$ANONYMIZATION_QUERIES" | psql -h "$NEW_DATABASE_ENDPOINT" -U openapparelregistry -t -A openapparelregistry -f -
+
+        # Run pg_dump
+        read -n 1 -s -r -p "Press any key to start pg_dump"
+        echo
+        DUMP_FILE="$NEW_DATABASE_ID.dump"
+        pg_dump -Fc -h "$NEW_DATABASE_ENDPOINT" -U openapparelregistry openapparelregistry > "$DUMP_FILE"
+
+        DELETE="aws rds delete-db-instance \
+            --no-paginate \
+            --db-instance-identifier $NEW_DATABASE_ID \
+            --skip-final-snapshot"
+        echo "Will run '$DELETE'"
+        read -n 1 -s -r -p "Press any key to delete $NEW_DATABASE_ID"
+        echo
+        $DELETE
+
+        # Run pg_restore
+        export PGPASSWORD="$STAGING_PGPASSWORD" 
+        PG_RESTORE="pg_restore -h $STAGING_DATABASE_ENDPOINT  -U openapparelregistry -d openapparelregistry --clean --if-exists --no-owner $DUMP_FILE"
```

Comment:
For OS Hub the username and database name will be `opensupplyhub` so we should change them here for the `ogr/develop` branch version.

Reply:
There's also `STAGING_ID` and `PRODUCTION_ID` at the top of the script, and we reference `openapparelregistry` in a few other places as well - do we want to set this up to expect an OS Hub staging _and_ production database (assuming one will exist soon)?

Reply:
Updated _all_ references of `openapparelregistry` to `opensupplyhub` in https://github.com/open-apparel-registry/open-apparel-registry/pull/1953/commits/df17bd774c75485fb9784efdb8e4f7345e3ef4b2

Reply:
it does make sense for the ogr/develop version of the script to use opensupplyhub everywhere. In the interim period where we use this script to restore OAR prod to OS Hub staging, we will need to override the variables accordingly. This is fine.